### ISSUE TITLE
Switchable Normalization ne

### ISSUE THREAD

**Issue by IPNUISTlegal** [switchablenorms/Switchable-Normalization#11]
![image](https://user-images.githubusercontent.com/35527568/47191595-f41fdd80-d37a-11e8-8f04-f08d369103c7.png)
mean_weight and var_weight both need softmax ，which assigns value to IN, LN, and BN . I tried the version of TensorFlow, which is more time-consuming and computerized than just BN. 
Am I right?
Thx!
